spec/acceptance/suites/rpm_docker/nodesets/default.yml

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
Why is this in beaker? It seems this is limiting how you can use Docker to solve this issue.